CONTRIBUTING.rst

```diff
@@ -104,15 +102,16 @@ git with your other code changes.
 Tests
 -----
 
-Currently, test coverage is very low. We're working on increasing this, and to
-make life easier we are using Travis CI (http://travis-ci.org/) for continuous
-integration. This means that the test suite is run every time a code change is
-made, so we can try and make sure we avoid basic bugs and other regressions.
+There is a CI/CD pipeline implemented using Github actions that covers code formatting, code security and unit tests.
+All PR's are required to pass the full test coverage suite before it can be merged to the main branch.
```

Comment:
We can also add that it will be reviewed by one of the maintainer before merging

Reply:
Added